pkgs/development/interpreters/starlark/default.nix

```diff
@@ -12,10 +12,18 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-lgL5o3MQfZekZ++BNESwV0LeoTxwEZfziQAe99zm4RY=";
 
+  patches = [
+    # Fix floating point imprecision issue in the test suite.
+    # https://github.com/google/starlark-go/pull/409
+    (fetchpatch {
+      url = "https://github.com/google/starlark-go/pull/409.patch";
```

Comment:
Please refer to the commit instead of the PR for fetching the patch

Reply:
I mean, you can refer to patches by commit hash rather than pull request ID.

E.g. https://github.com/bdd/starlark-go/commit/be6ed3bfcc376e5bf6fe2257ae89ddfb00d14e2c.patch